flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/schema/common/TableIdRouter.java

```diff
@@ -76,7 +77,7 @@ public List<TableId> route(TableId sourceTableId) {
         return routingCache.getUnchecked(sourceTableId);
     }
 
-    private List<TableId> calculateRoute(TableId sourceTableId) {
+    public List<TableId> calculateRoute(TableId sourceTableId) {
```

Comment:
Don't modify this, it's an internal method. Call `TableIdRouter#route` externally so the results could be cached.

Reply:
@aiwenmo  could you add some feedback about this comment？

Reply:
I've already replaced `TableIdRouter#calculateRoute` with `TableIdRouter#route`.I forgot to remove the change here.